### DISCUSSION TITLE
TODO: Issue templates

### DISCUSSION THREAD

**Discussion by freefd** [0x2142/frigate-notify#81] [General]
Defining an issue template with mandatory fields. Categories:

1. Bug
See example https://github.com/blakeblackshear/frigate/discussions/new?category=bug-report
2. Feature Request / Enhancement
```
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to ... so that I can ... 

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**
Add any other context or screenshots about the feature request here.
```
3. General
See example https://github.com/blakeblackshear/frigate/discussions/new?category=general-support

**Comment by 0x2142** [comment]
Yeah that's probably a good idea at this point. I honestly never expected anyone to really use this 😅

But I suppose it's time to set something like this up 👍
